2Fauth install fails

### ✅ Have you read and understood the above guidelines?

yes

### 📜 What is the name of the script you are using?

2FAuth

### 📂 What was the exact command used to execute the script?

bash -c "$(curl -fsSL https://raw.githubusercontent.com/community-scripts/ProxmoxVE/main/ct/2fauth.sh)"

### 📝 Provide a clear and concise description of the issue.

I get the following error when trying to install 2Fauth.

```
Fatal error: Uncaught Error: Class "Normalizer" not found in /usr/share/php/Symfony/Component/String/AbstractUnicodeString.php:31
Stack trace:
#0 /usr/share/php/Symfony/Component/Console/Helper/Helper.php(65): [constant expression]()
#1 /usr/share/php/Symfony/Component/Console/Application.php(855): Symfony\Component\Console\Helper\Helper::width()
#2 /usr/share/php/Symfony/Component/Console/Application.php(840): Symfony\Component\Console\Application->doRenderThrowable()
#3 /usr/share/php/Composer/Console/Application.php(401): Symfony\Component\Console\Application->renderThrowable()
#4 /usr/share/php/Symfony/Component/Console/Application.php(171): Composer\Console\Application->doRun()
#5 /usr/share/php/Composer/Console/Application.php(141): Symfony\Component\Console\Application->run()
#6 /usr/bin/composer(89): Composer\Console\Application->run()
#7 {main}
  thrown in /usr/share/php/Symfony/Component/String/AbstractUnicodeString.php on line 31
 ⠧
[ERROR] in line 67: exit code 0: while executing command $STD composer update --no-plugins --no-scripts

```

### ⚙️ What settings are you using?

- [x] Default Settings
- [ ] Advanced Settings

### 🖥️ Which Linux distribution are you using?

Debian 12

### 🔄 Steps to reproduce the issue.

Run Install script

`bash -c "$(curl -fsSL https://raw.githubusercontent.com/community-scripts/ProxmoxVE/main/ct/2fauth.sh)"`

Wait until it fails


### ❌ Paste the full error output (if available).

✔️  Installed Dependencies
  ✔️  Set up Database
 ⠼Loading composer repositories with package information
 ⠼Updating dependencies                               
 ⠴Lock file operations: 0 installs, 14 updates, 0 removals
  - Upgrading firebase/php-jwt (v6.11.0 => v6.11.1)
  - Upgrading laravel/pint (v1.21.2 => v1.22.0)
  - Upgrading laravel/serializable-closure (v2.0.3 => v2.0.4)
  - Upgrading laravel/socialite (v5.18.0 => v5.19.0)
  - Upgrading nesbot/carbon (3.8.6 => 3.9.0)
  - Upgrading nette/utils (v4.0.5 => v4.0.6)
  - Upgrading nunomaduro/collision (v8.7.0 => v8.8.0)
  - Upgrading phpunit/phpunit (11.5.15 => 11.5.17)
  - Upgrading spatie/laravel-package-tools (1.92.0 => 1.92.2)
  - Upgrading symfony/console (v7.2.1 => v7.2.5)
  - Upgrading symfony/error-handler (v7.2.4 => v7.2.5)
  - Upgrading symfony/http-foundation (v7.2.3 => v7.2.5)
  - Upgrading symfony/http-kernel (v7.2.4 => v7.2.5)
  - Upgrading symfony/process (v7.2.4 => v7.2.5)
Writing lock file
Installing dependencies from lock file (including require-dev)
Package operations: 149 installs, 0 updates, 0 removals
  - Downloading voku/portable-ascii (2.0.3)
  - Downloading symfony/polyfill-php80 (v1.31.0)
  - Downloading symfony/polyfill-mbstring (v1.31.0)
  - Downloading symfony/polyfill-ctype (v1.31.0)
  - Downloading phpoption/phpoption (1.9.3)
  - Downloading graham-campbell/result-type (v1.1.3)
  - Downloading vlucas/phpdotenv (v5.6.1)
  - Downloading symfony/css-selector (v7.2.0)
  - Downloading tijsverkoyen/css-to-inline-styles (v2.3.0)
  - Downloading symfony/var-dumper (v7.2.3)
  - Downloading symfony/polyfill-uuid (v1.31.0)
  - Downloading symfony/uid (v7.2.0)
  - Downloading symfony/deprecation-contracts (v3.5.1)
  - Downloading symfony/routing (v7.2.3)
  - Downloading symfony/process (v7.2.5)
  - Downloading symfony/polyfill-php83 (v1.31.0)
  - Downloading symfony/polyfill-intl-normalizer (v1.31.0)
  - Downloading symfony/polyfill-intl-idn (v1.31.0)
  - Downloading symfony/mime (v7.2.4)
  - Downloading psr/container (2.0.2)
  - Downloading symfony/service-contracts (v3.5.1)
  - Downloading psr/event-dispatcher (1.0.0)
  - Downloading symfony/event-dispatcher-contracts (v3.5.1)
  - Downloading symfony/event-dispatcher (v7.2.0)
  - Downloading psr/log (3.0.2)
  - Downloading doctrine/lexer (3.0.1)
  - Downloading egulias/email-validator (4.0.4)
  - Downloading symfony/mailer (v7.2.3)
  - Downloading symfony/http-foundation (v7.2.5)
  - Downloading symfony/error-handler (v7.2.5)
  - Downloading symfony/http-kernel (v7.2.5)
  - Downloading symfony/finder (v7.2.2)
  - Downloading symfony/polyfill-intl-grapheme (v1.31.0)
  - Downloading symfony/string (v7.2.0)
  - Downloading symfony/console (v7.2.5)
  - Downloading ramsey/collection (2.1.1)
  - Downloading brick/math (0.12.3)
  - Downloading ramsey/uuid (4.7.6)
  - Downloading psr/simple-cache (3.0.0)
  - Downloading nunomaduro/termwind (v2.3.0)
  - Downloading symfony/translation-contracts (v3.5.1)
  - Downloading symfony/translation (v7.2.4)
  - Downloading psr/clock (1.0.0)
  - Downloading symfony/clock (v7.2.0)
  - Downloading carbonphp/carbon-doctrine-types (3.2.0)
  - Downloading nesbot/carbon (3.9.0)
  - Downloading monolog/monolog (3.9.0)
  - Downloading psr/http-message (2.0)
  - Downloading psr/http-factory (1.1.0)
  - Downloading league/uri-interfaces (7.5.0)
  - Downloading league/uri (7.5.1)
  - Downloading league/mime-type-detection (1.16.0)
  - Downloading league/flysystem-local (3.29.0)
  - Downloading league/flysystem (3.29.1)
  - Downloading nette/utils (v4.0.6)
  - Downloading nette/schema (v1.3.2)
  - Downloading dflydev/dot-access-data (v3.0.3)
  - Downloading league/config (v1.2.0)
  - Downloading league/commonmark (2.6.1)
  - Downloading laravel/serializable-closure (v2.0.4)
  - Downloading laravel/prompts (v0.3.5)
  - Downloading guzzlehttp/uri-template (v1.0.4)
  - Downloading psr/http-client (1.0.3)
  - Downloading ralouphie/getallheaders (3.0.3)
  - Downloading guzzlehttp/psr7 (2.7.1)
  - Downloading guzzlehttp/promises (2.2.0)
  - Downloading guzzlehttp/guzzle (7.9.3)
  - Downloading fruitcake/php-cors (v1.3.0)
  - Downloading webmozart/assert (1.11.0)
  - Downloading dragonmantank/cron-expression (v3.4.0)
  - Downloading doctrine/inflector (2.0.10)
  - Downloading laravel/framework (v11.44.2)
  - Downloading composer/pcre (3.3.2)
  - Downloading composer/class-map-generator (1.6.1)
  - Downloading barryvdh/reflection-docblock (v2.3.1)
  - Downloading barryvdh/laravel-ide-helper (v3.5.5)
  - Downloading sebastian/environment (7.2.0)
  - Downloading staabm/side-effects-detector (1.0.5)
  - Downloading sebastian/version (5.0.2)
  - Downloading sebastian/type (5.1.2)
  - Downloading sebastian/recursion-context (6.0.2)
  - Downloading sebastian/object-reflector (4.0.1)
  - Downloading sebastian/object-enumerator (6.0.1)
  - Downloading sebastian/global-state (7.0.2)
  - Downloading sebastian/exporter (6.3.0)
  - Downloading sebastian/diff (6.0.2)
  - Downloading sebastian/comparator (6.3.1)
  - Downloading sebastian/code-unit (3.0.3)
  - Downloading sebastian/cli-parser (3.0.2)
  - Downloading phpunit/php-timer (7.0.1)
  - Downloading phpunit/php-text-template (4.0.1)
  - Downloading phpunit/php-invoker (5.0.1)
  - Downloading phpunit/php-file-iterator (5.1.0)
  - Downloading theseer/tokenizer (1.2.3)
  - Downloading nikic/php-parser (v5.4.0)
  - Downloading sebastian/lines-of-code (3.0.1)
  - Downloading sebastian/complexity (4.0.1)
  - Downloading sebastian/code-unit-reverse-lookup (4.0.1)
  - Downloading phpunit/php-code-coverage (11.0.9)
  - Downloading phar-io/version (3.2.1)
  - Downloading phar-io/manifest (2.0.4)
  - Downloading myclabs/deep-copy (1.13.0)
  - Downloading phpunit/phpunit (11.5.17)
  - Downloading jean85/pretty-package-versions (2.1.1)
  - Downloading fidry/cpu-core-counter (1.2.0)
  - Downloading brianium/paratest (v7.8.3)
  - Downloading chillerlan/php-settings-container (3.2.1)
  - Downloading chillerlan/php-qrcode (5.0.3)
  - Downloading doctormckay/steam-totp (v1.0.1)
  - Downloading enshrined/svg-sanitize (0.20.0)
  - Downloading fakerphp/faker (v1.24.1)
  - Downloading google/protobuf (v4.30.2)
  - Downloading jackiedo/path-helper (v1.0.0)
  - Downloading jackiedo/dotenv-editor (dev-master e8743b3)
  - Downloading mobiledetect/mobiledetectlib (2.8.45)
  - Downloading jaybizzle/crawler-detect (v1.3.4)
  - Downloading jenssegers/agent (v2.6.4)
  - Downloading khanamiryan/qrcode-detector-decoder (2.0.2)
  - Downloading laragear/meta-model (v1.1.1)
 ⠦  - Downloading laragear/webauthn (v2.0.3)
  - Downloading phpstan/phpstan (1.12.23)
  - Downloading iamcal/sql-parser (v0.5)
  - Downloading larastan/larastan (v2.10.0)
  - Downloading symfony/psr-http-message-bridge (v7.2.0)
  - Downloading paragonie/random_compat (v9.99.100)
  - Downloading paragonie/constant_time_encoding (v3.0.0)
  - Downloading phpseclib/phpseclib (3.0.43)
  - Downloading nyholm/psr7 (1.8.2)
  - Downloading league/event (2.3.0)
  - Downloading lcobucci/jwt (5.5.0)
  - Downloading lcobucci/clock (3.3.1)
  - Downloading defuse/php-encryption (v2.4.0)
  - Downloading league/oauth2-server (8.5.5)
  - Downloading firebase/php-jwt (v6.11.1)
  - Downloading laravel/passport (v12.4.2)
  - Downloading laravel/pint (v1.22.0)
  - Downloading psy/psysh (v0.12.8)
  - Downloading laravel/tinker (v2.10.1)
  - Downloading laravel/ui (v4.6.1)
  - Downloading league/oauth1-client (v1.11.0)
  - Downloading hamcrest/hamcrest-php (v2.0.1)
  - Downloading mockery/mockery (1.6.12)
  - Downloading filp/whoops (2.18.0)
  - Downloading nunomaduro/collision (v8.8.0)
  - Downloading laravel/socialite (v5.19.0)
  - Downloading socialiteproviders/manager (v4.8.1)
  - Downloading spatie/laravel-package-tools (1.92.2)
  - Downloading spatie/eloquent-sortable (4.4.2)
  - Downloading spomky-labs/otphp (11.3.0)

PHP Fatal error:  Uncaught Error: Class "Normalizer" not found in /usr/share/php/Symfony/Component/String/AbstractUnicodeString.php:31
Stack trace:
#0 /usr/share/php/Symfony/Component/Console/Helper/Helper.php(65): [constant expression]()
#1 /usr/share/php/Symfony/Component/Console/Application.php(855): Symfony\Component\Console\Helper\Helper::width()
#2 /usr/share/php/Symfony/Component/Console/Application.php(840): Symfony\Component\Console\Application->doRenderThrowable()
#3 /usr/share/php/Composer/Console/Application.php(401): Symfony\Component\Console\Application->renderThrowable()
#4 /usr/share/php/Symfony/Component/Console/Application.php(171): Composer\Console\Application->doRun()
#5 /usr/share/php/Composer/Console/Application.php(141): Symfony\Component\Console\Application->run()
#6 /usr/bin/composer(89): Composer\Console\Application->run()
#7 {main}
  thrown in /usr/share/php/Symfony/Component/String/AbstractUnicodeString.php on line 31

Fatal error: Uncaught Error: Class "Normalizer" not found in /usr/share/php/Symfony/Component/String/AbstractUnicodeString.php:31
Stack trace:
#0 /usr/share/php/Symfony/Component/Console/Helper/Helper.php(65): [constant expression]()
#1 /usr/share/php/Symfony/Component/Console/Application.php(855): Symfony\Component\Console\Helper\Helper::width()
#2 /usr/share/php/Symfony/Component/Console/Application.php(840): Symfony\Component\Console\Application->doRenderThrowable()
#3 /usr/share/php/Composer/Console/Application.php(401): Symfony\Component\Console\Application->renderThrowable()
#4 /usr/share/php/Symfony/Component/Console/Application.php(171): Composer\Console\Application->doRun()
#5 /usr/share/php/Composer/Console/Application.php(141): Symfony\Component\Console\Application->run()
#6 /usr/bin/composer(89): Composer\Console\Application->run()
#7 {main}
  thrown in /usr/share/php/Symfony/Component/String/AbstractUnicodeString.php on line 31
 ⠧
[ERROR] in line 67: exit code 0: while executing command $STD composer update --no-plugins --no-scripts

### 🖼️ Additional context (optional).

_No response_

--- a/install/2fauth-install.sh
+++ b/install/2fauth-install.sh
@@ -16,18 +16,18 @@ update_os
 msg_info "Installing Dependencies"
 
 $STD apt-get install -y \
-        lsb-release \
-        gpg
+  lsb-release \
+  gpg
 
 curl -fsSL https://packages.sury.org/php/apt.gpg | gpg --dearmor -o /usr/share/keyrings/deb.sury.org-php.gpg
-echo "deb [signed-by=/usr/share/keyrings/deb.sury.org-php.gpg] https://packages.sury.org/php/ $(lsb_release -sc) main" > /etc/apt/sources.list.d/php.list
+echo "deb [signed-by=/usr/share/keyrings/deb.sury.org-php.gpg] https://packages.sury.org/php/ $(lsb_release -sc) main" >/etc/apt/sources.list.d/php.list
 $STD apt-get update
 
 $STD apt-get install -y \
-        nginx \
-        composer \
-        php8.3-{bcmath,common,ctype,curl,fileinfo,fpm,gd,mbstring,mysql,xml,cli} \
-        mariadb-server
+  nginx \
+  composer \
+  php8.3-{bcmath,common,ctype,curl,fileinfo,fpm,gd,intl,mbstring,mysql,xml,cli} \
+  mariadb-server
 msg_ok "Installed Dependencies"
 
 msg_info "Setting up Database"
@@ -38,10 +38,10 @@ $STD mysql -u root -e "CREATE DATABASE $DB_NAME;"
 $STD mysql -u root -e "CREATE USER '$DB_USER'@'localhost' IDENTIFIED WITH mysql_native_password AS PASSWORD('$DB_PASS');"
 $STD mysql -u root -e "GRANT ALL ON $DB_NAME.* TO '$DB_USER'@'localhost'; FLUSH PRIVILEGES;"
 {
-        echo "2FAuth Credentials"
-        echo "Database User: $DB_USER"
-        echo "Database Password: $DB_PASS"
-        echo "Database Name: $DB_NAME"
+  echo "2FAuth Credentials"
+  echo "Database User: $DB_USER"
+  echo "Database Password: $DB_PASS"
+  echo "Database Name: $DB_NAME"
 } >>~/2FAuth.creds
 msg_ok "Set up Database"
 
@@ -56,12 +56,12 @@ cp .env.example .env
 IPADDRESS=$(hostname -I | awk '{print $1}')
 
 sed -i -e "s|^APP_URL=.*|APP_URL=http://$IPADDRESS|" \
-        -e "s|^DB_CONNECTION=$|DB_CONNECTION=mysql|" \
-        -e "s|^DB_DATABASE=$|DB_DATABASE=$DB_NAME|" \
-        -e "s|^DB_HOST=$|DB_HOST=127.0.0.1|" \
-        -e "s|^DB_PORT=$|DB_PORT=3306|" \
-        -e "s|^DB_USERNAME=$|DB_USERNAME=$DB_USER|" \
-        -e "s|^DB_PASSWORD=$|DB_PASSWORD=$DB_PASS|" .env
+  -e "s|^DB_CONNECTION=$|DB_CONNECTION=mysql|" \
+  -e "s|^DB_DATABASE=$|DB_DATABASE=$DB_NAME|" \
+  -e "s|^DB_HOST=$|DB_HOST=127.0.0.1|" \
+  -e "s|^DB_PORT=$|DB_PORT=3306|" \
+  -e "s|^DB_USERNAME=$|DB_USERNAME=$DB_USER|" \
+  -e "s|^DB_PASSWORD=$|DB_PASSWORD=$DB_PASS|" .env
 
 export COMPOSER_ALLOW_SUPERUSER=1
 $STD composer update --no-plugins --no-scripts

